### DISCUSSION TITLE
Add a factor for non-symptomatic (hidden) cases to model?

### DISCUSSION THREAD

**Discussion by schaupets** [neherlab/covid19_scenarios#623] [General]
Suggestion: extend the state model (https://covid19-scenarios.org/about) with a factor X for non-symptomatic hidden cases.

Motivation: there is a lot of ongoing discussions, if these hidden cases could reduce the peak on Hospitalized/criticial/ICU patients significantly, 
so that the lockdown measures can be reduced. 

Example: I did a simple Excel-VBA simulation, which showed that a hidden factor of 10 could keep the ICU maximum below the number of beds, 
even if R0 is larger than 1,0 and we go thru the full peak of immunisation. 
I used this "hidden factor" as follows: "HiddenInfected = VisibleInfected * factor" and "susceptible = population - VisibleInfected - HiddenInfected"

Goal: Would be interesting, what the much better Covid19 Scenario Simulation will compute with such a hidden factor. 

What do you think?

**Comment by rneher** [comment]
Thanks for the suggestions. At some level, we already have this. If you look at the (collapsed) severity table. 
![image](https://user-images.githubusercontent.com/8379168/79957444-deb0d580-8481-11ea-8a86-faccced13e7c.png)

we assume that the statistic only catches 5% of children and that only a small fraction of them get ill. Does this address your point?

**Comment by schaupets** [comment]
My point is this: your model parameters are tuned with the existing "data", see "(data)" lines in the diagram.
But the suspicion is, that the data, i.e known cases are not all cases that had corona and many more people already had corona and recovered without been in the statisitics.
They are not Susceptible anymore, at least not in 2020. 
Not including these hidden cases leads to the effect that the **Susceptible (S) number** is too high in your current model and therefore the prediction wrong, independent from age group. 

They are doing studies here in Munich to find out, how much such "dark/hidden" cases we have:
https://www.lmu-klinikum.de/aktuelles/pressemitteilungen/munchner-tropeninstitut-beginnt-stichprobenanalyse-zur-verbreitung-der-corona-pandemie-und-zur-wirksamkeit-von-gegenmassnahmen/6afa2c06cb6745a9

The interessting question is, how much the curve will flatten because of this effect, i.e. the lower Susceptible number. Independent from age group.
A lower Susceptible number will lower the effective R0. A Susceptible percentage below 30% stops the pandemie, at least in most models. 

So its all about the real, true Susceptible number. 
Did this make point clearer?

**Reply by nnoll** [reply]
Hey @schaupets, 

I think @rneher was referring to the "confirmed" column of the severity table. If you adjust this to be a lower number, the model will run with a higher number of "dark" cases as they will progress immediately from the infected state to the recovered state without progressing towards severe. So if one wanted to look at this effect, indepedent of age, one would adjust all these numbers to be lower by a uniform scale.

Additionally, it is important to note that while we do fit to the data (both infections and fatalities) we do estimate the "under-reporting" fraction in the process. However, this is done assuming we capture population fatality statistics accurately, which is most likely a faulty assumption. Despite this, we ultimately just use these estimates to populate our initial guess of the parameters. You can change them all in the UI.

Hope this answers your question!

**Reply by schaupets** [reply]
I think that I understand now your table with the "Confirmed% of total" colum:  The column "Confirmed" reflects our assumption on what fraction of infections are reflected in the statistics
And yes -I must correct myself -  the underestimated cases are probably age dependend, as your table indicates.

To understand you correctly: Does the simulation add 95 hidden cases for each confirmed case in the age group 10-19 ?  
When I run the simulation for Germany, I see at 2020-04-22:  
145k cumulative cases =  corresponds to worldometer confirmed cases (data) and
882k recovered - which is factor 10 above the worldometer data and factor 6 higher than cumulative cases. 
So 6 times more recovered than cases? Seems funny.  This is the hidden factor?

As you see from my question, this behaviour is OK but is not very obvious in the grafik and simulation results.

BTW: A factor of 10-15% confirmed in age 20-39 seems a little bit too optimistic, given the results from the Roosevelt: 
Bisherigen Studien zufolge treten laut Hyten bei 20 bis 50 Prozent der Infizierten keine Symptome auf. An Bord der „USS Theodore Roosevelt“ zeigten hingegen „fast 60, 70 Prozent“ der Infizierten keine Symptome, sagte Hyten.
https://www.faz.net/aktuell/politik/ausland/uss-theodore-roosevelt-zwei-drittel-der-infizierten-ohne-symptome-16731323.html?utm_source=pocket-newtab

**Reply by rneher** [reply]
What is considered asymptomatic and mild is a little fuzzy. When we fit to the data, we put most weight on the fatalities count and less on the cases, for which we include a factor by which they are underreported. This factor tends to be about 1/5 - 1/10.

**Comment by kaiserso** [comment]
I was also trying to understand how the model handles asymptomatic or sub-clinical infection, which seems to be a key (but poorly characterized) determinant of epidemic progression. Currently it appears to treat non-detection and sub-clinical interchangeably in terms of progression, hospitalization and death. However, as there is not a separate compartment for asymptomatic or subclinical infection, I believe these infections are not distinguished in terms of infectiousness in the model. While there is ample evidence for asymptomatic or presymtomatic transmission, it seems extreme to assume 1:1 ratio of infectiousness between asymptomatic and symptomatic infection.   If this is correct it would cause asymptomatic cases to account disproportionately for transmission events. Perhaps this is not critical currently because the model does not provide outputs in terms of mild vs asymptomatic?

**Reply by rneher** [reply]
These are all valid points, but these distinctions between pre-symptomatic, asymptomatic, and mild are not so critical since our main object is to project deaths and hospital demand. The latter are much better understood and adding more parameters to the model likely adds more confusion than clarity.

**Reply by kaiserso** [reply]
Thank you for your response. I agree in an unmitigated epidemic it does not impact hospitalization and mortality rates, however I think it would matter for predicting impact of mitigation strategies based on symptoms such as self-isolation - but as you point out, not the focus of this model.
